cmd/kube-proxy/app/server.go

```diff
@@ -142,7 +143,7 @@ type Options struct {
 func (o *Options) AddFlags(fs *pflag.FlagSet) {
 	o.addOSFlags(fs)
 
-	fs.StringVar(&o.ConfigFile, "config", o.ConfigFile, "The path to the configuration file.")
+	fs.StringVar(&o.ConfigFile, proxyConfigArg, o.ConfigFile, "The path to the kube-proxy configuration file, which adheres the the kube-proxy API configuration object.")
```

Comment:
nit: "which adheres the" seems to be pretty awkward wording

Reply:
There are also two "the"s there.
It would be nice if we could link to the API documentation but it looks like https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/ doesn't cover that...